reliability-engineering/terraform/modules/concourse-monitoring/files/prometheus-init.sh

```diff
@@ -60,6 +60,19 @@ scrape_configs:
       - source_labels: [__meta_ec2_instance_id]
         target_label: instance
 
+  - job_name: concourse_grafana
+    metrics_path: '/metrics'
+    scheme: 'https'
```

Comment:
is it really over `https` if it's on the internal network? ... I would have thought it was `http` unless it was going via the public load-balancer.

Reply:
Just realised that https is from concourse_monitoring_http lb. concourse_grafana lb has http. I will drop https.

Reply:
I have replaced https with http.